### PR TITLE
Update USAGE

### PR DESCRIPTION
**Pull request by geigerj**
Corresponds to new config names in googleapis

Pre-push hook installed.

Change-Id: I1c0826ab5f5a3649068e489b24941a31cdffa79c

### CODE REVIEW

**Comment by geigerj** [comment]
See googleapis/googleapis#9

File: USAGE.rst [hunk 1]
```diff
@@ -66,8 +66,8 @@ Config generation
   ::
 
      python execute_pipeline.py \
-        --config "../googleapis/pipeline_config/pipeline_logging.yaml:logging_common,\
-        ../googleapis/pipeline_config/pipeline_common.yaml:default" \
+        --config ""../googleapis/gapic/api/artman_logging.yaml:logging_common,\
```
Comment (garrettjonesgoogle) [inline USAGE.rst]: is the double double quote a mistake?

Comment (geigerj) [inline USAGE.rst]: Yes, now fixed

**Comment by tbetbetbe** [comment]
LGTM

**Pull request merged**